### PR TITLE
Blaze: Add unit tests for destination parameters screen

### LINKED ISSUE #10827: Blaze: Add unit tests for the parameters screen

### PR DESCRIPTION
**Pull request by 0nko**
Fixes #10827.

**To test:**
Making sure all lights are green is fine.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-android/milestone/202">17.9</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/src/test/kotlin/com/woocommerce/android/ui/blaze/creation/destination/BlazeCampaignCreationAdDestinationParametersViewModelTest.kt [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    fun setup(params: Map<String, String> = emptyMap()) {
+        viewModel = BlazeCampaignCreationAdDestinationParametersViewModel(
+            BlazeCampaignCreationAdDestinationParametersFragmentArgs(
+                DestinationParameters(targetUrl, params)
+            ).toSavedStateHandle()
+        )
+    }
+
+    @Test
+    fun `onBackPressed triggers correct event`() = testBlocking {
```
Comment (hichamboushaba) [inline WooCommerce/src/test/kotlin/com/woocommerce/android/ui/blaze/creation/destination/BlazeCampaignCreationAdDestinationParametersViewModelTest.kt]: minor np, the tests names should follow our conventions https://github.com/woocommerce/woocommerce-android/blob/fd7e0f18cac5d7f44f02892253a0f7cd961721a3/CONVENTION.md#naming-1

**Review by hichamboushaba — APPROVED** [review]
Nice work @0nko, I left one np comment, I'll leave it to you to decide if you want to apply it or ignore it.

File: WooCommerce/src/test/kotlin/com/woocommerce/android/ui/blaze/creation/destination/BlazeCampaignCreationAdDestinationParametersViewModelTest.kt (hunk 1 above)
Comment (0nko) [reply WooCommerce/src/test/kotlin/com/woocommerce/android/ui/blaze/creation/destination/BlazeCampaignCreationAdDestinationParametersViewModelTest.kt]: You're right, thanks! I've updated the names.

Comment (hichamboushaba) [reply WooCommerce/src/test/kotlin/com/woocommerce/android/ui/blaze/creation/destination/BlazeCampaignCreationAdDestinationParametersViewModelTest.kt]: Thank you @0nko

**Pull request merged**